examples/trusted_information_retrieval/config/config.toml

```diff
@@ -1 +1,9 @@
-database_url = "https://localhost:8888"
+name = "trusted_information_retrieval"
+
+[modules]
+app = { path = "examples/target/wasm32-unknown-unknown/release/trusted_information_retrieval.wasm" }
+database_proxy = { path = "examples/target/wasm32-unknown-unknown/release/database_proxy.wasm" }
+
+[initial_node_configuration]
+wasm_module_name = "app"
+wasm_entrypoint_name = "oak_main"
```

Comment:
This seems like it's repeating everywhere... do we actually need this? Can we just have this as default and only have it when it's different at least?

Reply:
I suggest to stick with this for now, we can factor it out if it gets annoying in the future, but it's another piece of documentation that developers need to find (what is the default module name? what should the function be called? etc.)